sensor_msgs/PointCloud2[] point_cloud2_vec
